test/plugins/wasi_llm/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+wasmedge_add_executable(WasiLLMTests
+  wasi_llm.cpp
+)
+
+add_dependencies(WasiLLMTests
+  wasmedgePluginWasiLLM
+)
+
+target_include_directories(WasiLLMTests
+	PUBLIC
+	$<TARGET_PROPERTY:wasmedgePlugin,INCLUDE_DIRECTORIES>
+  $<TARGET_PROPERTY:wasmedgePluginWasiLLM,INCLUDE_DIRECTORIES>
+)
+
+target_link_libraries(WasiLLMTests
+  PRIVATE
+  ${GTEST_BOTH_LIBRARIES}
+)
+
+# Link to the WasmEdge library
+if(WASMEDGE_LINK_PLUGINS_STATIC)
+  target_link_libraries(WasiLLMTests
+    PRIVATE
+    wasmedgeCAPI
+  )
+else()
+  target_link_libraries(WasiLLMTests
+    PRIVATE
+    wasmedge_shared
+  )
+endif()
+
+function(download URL OUTPUT) #HASH)
+  file(DOWNLOAD
+    ${URL}
+    ${OUTPUT}
+    SHOW_PROGRESS
+    # EXPECTED_HASH ${HASH}
+  )
+endfunction()
+
+message(STATUS "Downloading GPT2 model check point to ${CMAKE_CURRENT_BINARY_DIR}/gpt2_124M.bin")
+download(
+  #https://huggingface.co/CompVis/stable-diffusion-v-1-4-original/resolve/main/sd-v1-4.ckpt
```

Comment:
we need somewhere to store these `.bin` files, where should we put them? @hydai

Reply:
Is this file generated by some tools? Or is it from a standalone repository or somewhere?

Reply:
i mean the checkpoint binary, training and validation data, tokenizer data... they are generated by scripts in `llm.c`, but I would like use rebuilt ones for sake of simplicity.

Reply:
more specifically, https://github.com/karpathy/llm.c/blob/master/train_gpt2.py

Reply:
It seems like there is a download script for these files. I would like to know if these files will change due to the version upgrade of llm.c or not. If it will always be the same, then one solution is using this file instead. Otherwise, we can have a huggingface repo for this.
https://github.com/karpathy/llm.c/blob/master/dev/download_starter_pack.sh

Reply:
thanks, let's just grab these from huggingface